common/src/test/java/io/pravega/common/util/RetryTests.java

```diff
@@ -255,6 +254,11 @@ public void retryIndefiniteTest() throws ExecutionException, InterruptedExceptio
                     }
                 }), pool).get();
         assert i.get() == 10;
+        try {
+            Retry.indefinitelyWithExpBackoff(0, 0, 0, e -> i.getAndIncrement());
```

Comment:
Use `AssertThrows` instead of this try/catch + assertion construct.

Reply:
Added the `AssertThrows` and removed `try/catch`